frontend/src/App.js

```diff
@@ -1,23 +1,53 @@
-import logo from './logo.svg';
-import './App.css';
+import {useEffect, useReducer, useState} from 'react'
+import countriesReducer from './reducer/countriesReducer'
+
+import Header from './components/Header'
+import Footer from './components/Footer'
+import CountryDropdowns from './components/CountryDropdowns'
+import SymmetricCountryChart from './components/SymmetricCountryChart'
+import GlobalStyles from "./GlobalStyles";
 
 function App() {
+
+  const [countries, setCountries] = useReducer(
+    countriesReducer,
+    {data: [], isLoading: false, isError: false}
+  )
+  
+  const [displayedCountries, setDisplayedCountries] = useState({
+    countryLeft: {},
+    countryRight: {}
+  })
+  
+  useEffect(() => {
+    setCountries({type: 'COUNTRIES_FETCH_INIT'})
+    fetch('http://countrycheck.local/countries')
+      .then(response => response.json())
+      .then(result => {
+          setCountries({
+          type: 'COUNTRIES_FETCH_SUCCESS',
+          payload: result
+          })
```

Comment:
Looks like your code formatter is not working here. Maybe it's an indicator for another issue?